packages/docs/docs/getting-started/what-can-you-build-with-unirep.md

```diff
@@ -0,0 +1,29 @@
+
+Peer-to-Peer Anonymous Lending:
+ - A problem in current decentralized protocols is that most of them rely on the credit system being over-collateralized. With UniRep, the reputation value will be the credit-score of a prospective borrower that will change based on if they follow the credit rules of the prospective system.
+
+Reddit built with UniRep:
+- This example may be easiest to understand. The most important reputation value in Reddit is the karma user's earn. If we build Reddit with UniRep, the user's get to earn their karma anonymously while participtating in various different forums. You can then build in other "levels" into the platform when X karma gives you access to "Y", or grant a user the "mod" status or something of this nature.
+
+Anonymous GitHub:
+- Some developers may prefer to publish code anonymously. In A GitHub-type application built with UniRep, the reputation values could be the amount of commits a developer makes a repository, the amount of stars they have collectively on their profile, and/or if they have made successful PRs on a project. These reputation values can then be used to grant access to the anonymous developer to push to the main branch.
+
+Eth or Token Attesters:
+- It is possible to create an attester where the reputation value is Eth or some Token. This would allow the transfer of value anonymously. 
+
+Multi-Attester Applications:
+- It is possible to build attesters that interact with each other. For instance, a social media application can have multiple different groups. Let's say group X has their own reputation value called "Doge" and group Y has their reputation value called "Elon". The rules of how to earn these reputation values are defined by the developer. Multi-attester applications can interact with each other in various ways, too. If a member from group Y earns 50 "Elon" then they could be granted access to another attester that must have at least 50 "Doge" to join. The rules become limitless at this point, but bear in mind all the ideas possible with UniRep; social media is only one example.
+
+Medical Records:
+- A problem in the Electronic Health Records industry is interoperability between different healthcare providers. If you go to Hospital X because you live in California, they will store your records in their system. If you go to Hospital Y in New York because you get seriously injured, they have no idea about your medical background because of poor sharing of records. With UniRep, a user could have their medical records be attested to them be each provider (each provider would be an attester in this case) and submit a zk-proof  that they are the owner of that data when a provider needs access to it. Aside from interoperability, this would also have the user always be in control of their records. Please note: this is only an example and we must assume new cryptography, like zero-knowledge proofs, could be broken in 50 years.
```

Comment:
'A problem in the Electronic Health Records industry is the lack of interoperability between different healthcare providers. If you go to Hospital X because you live in California, they will store your records in their system. If you go to Hospital Y in New York because you get seriously injured, they will have no idea about your medical background because of poor (or no) systems for sharing records. Even the sharing of records between providers in the same city can come with long delays and major hassles for patients. With UniRep, a user's medical records could be attested to them by each provider at the time of service. When a provider required access to a record, the user would prove with zk that the requested data is theirs. The user would always be in control of their own records, and any stored medical data would be anonymous. In this case each provider would be an attester in an interoperable multi-attester system. Please note: this is only an example, and in dealing with sensitive data such as medical histories, we must keep in mind that new cryptography (like zero-knowledge proofs) may be broken in 50 years.'